README.md

```diff
@@ -129,15 +129,15 @@ file):
 
 ```
 docker build -t get_together .
-docker run -e "DEBUG_MODE=True" -e "SECRET_KEY=xxxxx" -e "ALLOWED_HOSTS=localhost,127.0.0.1" -d --name get_together -p 8000:8000 get_together
+docker run -e "DEBUG_MODE=True" -e "SECRET_KEY=xxxxx" -e "ALLOWED_HOSTS=localhost,127.0.0.1,0.0.0.0" -d --name get_together -p 8000:8000 get_together
 docker exec -it get_together venv/bin/python manage.py createsuperuser
 ```
 
 ### Using docker-compose
 
 ```
 docker-compose up -d
-docker-compose exec get_together python3 manage.py createsuperuser
+docker-compose exec get_together venv/bin/python manage.py createsuperuser
```

Comment:
I think you should use `env/bin/python` instead.

Reply:
The setup is a [bit different in the Dockerfile](https://github.com/GetTogetherComm/GetTogether/blob/5097e663a50a80b1fb8d9156968971c712c75c5e/Dockerfile#L7); I could just pass in the manage.py command as [`entrypoint` is being set in the Dockerfile](https://github.com/GetTogetherComm/GetTogether/blob/5097e663a50a80b1fb8d9156968971c712c75c5e/Dockerfile#L24).

I think the bigger issue is the [`-relocatable` switch](https://github.com/GetTogetherComm/GetTogether/blob/5097e663a50a80b1fb8d9156968971c712c75c5e/Dockerfile#L11), which is something that is [being deprecated](https://virtualenv.pypa.io/en/stable/userguide/#making-environments-relocatable)

Reply:
Oh, you're right, I didn't see the Dockerfile... sorry.